landing/index.html

```diff
@@ -19,7 +19,7 @@
                 <h2>veraPDF for WCAG</h2>
                 <span>Latest version: 0.1.0</span>
                 <span>Licence: GNU Lesser General Public License</span>
-                <button class="btn">Go to demo</button>
+                <a class="btn" href="./new-job/files">Go to demo</a>
```

Comment:
For the demo app let's use path `http://localhost/demo/new-job/files`. Note the `demo` subfolder.
That should simplify your nginx config drastically.
Note that you will need to define `homepage` in your `package.json` so that the context folder was not root for the prod version of the app.

Reply:
And for the link here let's simply pass `./demo`. The redirect in app should then pass us to the correct view (and that could be changed in one place in app if needed, without affecting this link)